addons/medical_treatment/stringtable.xml

```diff
@@ -4491,22 +4491,35 @@
             <Chinese>從&lt;br/&gt;%2卸載&lt;br/&gt;%1</Chinese>
             <Chinesesimp>从&lt;br/&gt;%2卸载&lt;br/&gt;%1</Chinesesimp>
         </Key>
-        <Key ID="STR_ACE_Medical_Treatment_PlaceInBodyBag">
-            <English>Place body in bodybag</English>
-            <Spanish>Colocar cuerpo en bolsa para cadáveres</Spanish>
-            <Russian>Поместить тело в мешок для трупов</Russian>
-            <German>Körper in Leichesack verstauen</German>
-            <Polish>Zapakuj ciało do worka na zwłoki</Polish>
-            <French>Mettre le corps dans la housse mortuaire</French>
-            <Hungarian>Test hullazsákba helyezése</Hungarian>
-            <Italian>Metti il corpo nella sacca per cadaveri</Italian>
-            <Portuguese>Colocar corpo dentro do saco para cadáver</Portuguese>
-            <Czech>Umístit tělo do pytle na mrtvoly</Czech>
-            <Japanese>死体袋に入れる</Japanese>
-            <Korean>시체 운반용 부대에 담기</Korean>
-            <Chinesesimp>将尸体装入尸袋</Chinesesimp>
-            <Chinese>將屍體放入屍袋</Chinese>
-            <Turkish>Vücudu ceset torbasına yerleştir</Turkish>
+        <Key ID="STR_ACE_Medical_Treatment_PlaceInBodyBagBlack">
+            <English>Place body in black bodybag</English>
```

Comment:
Why not "Place body in bodybag (color)"?

Reply:
I did that at first, but then to me this seemed more natural. I see that for spray paint its a mix between the two formats.